Tests/FoundationTests/NSAttributedStringExtensionsTests.swift

```diff
@@ -306,6 +306,43 @@ final class NSAttributedStringExtensionsTests: XCTestCase {
         #endif
     }
 
+    // MARK: - Initializer
+    
+    #if canImport(UIKit)
```

Comment:
The imports of the tests should match the imports of the original functions exactly.